### ISSUE TITLE
Reserve for documentation

### ISSUE THREAD

**Issue by mjstealey** [mjstealey/wordpress-nginx-docker#1]
Reserved for documentation

**Comment by mjstealey** [comment]
### Installation screenshots

Select language
<img width="80%" alt="Select language" src="https://user-images.githubusercontent.com/5332509/44045885-f47a89fe-9ef7-11e8-8dae-0df0bfb269de.png">

Create admin user
<img width="80%" alt="Create admin user" src="https://user-images.githubusercontent.com/5332509/44045887-f4897cfc-9ef7-11e8-89c6-cfc96cfc9ca0.png">

Success
<img width="80%" alt="Success" src="https://user-images.githubusercontent.com/5332509/44045888-f49b344c-9ef7-11e8-9d65-39517f521d85.png">

First login
<img width="80%" alt="First login" src="https://user-images.githubusercontent.com/5332509/44045889-f4a71992-9ef7-11e8-8f5d-8ab16da481c2.png">

Site dashboard
<img width="80%" alt="Site dashboard" src="https://user-images.githubusercontent.com/5332509/44045890-f4b4b264-9ef7-11e8-935b-cbc546cd9e00.png">

View site
<img width="80%" alt="View site" src="https://user-images.githubusercontent.com/5332509/44045891-f4c5f90c-9ef7-11e8-88e4-fc8cfb61ea7d.png">

**Comment by behinam** [comment]
Hello
when i run i get this error:

wordpress    | MySQL Connection Error: (2002) Connection refused
wordpress    |
wordpress    | Warning: mysqli::__construct(): (HY000/2002): Connection refused in Standard input code on line 22
 my yml file

`version: '2'
services:
  nginx:
    image: nginx:latest
    container_name: nginx
    ports:
      - '80:80'
      - '443:443'
    volumes:
      - ./nginx:/etc/nginx/conf.d
      - ./logs/nginx:/var/log/nginx
      - ./wordpress:/var/www/html
      - ./certs:/etc/letsencrypt
      - ./certs-data:/data/letsencrypt
    links:
      - wordpress
    restart: always

  mysql:
    image: mariadb
    container_name: mysql
    ports:
      - '3306:3306'
    volumes:
      - ./mysql:/var/lib/mysql
    environment:
      - MYSQL_ROOT_PASSWORD=123456789
    restart: always

  wordpress:
    image: wordpress:php7.2-fpm
    container_name: wordpress
    volumes:
      - ./wordpress:/var/www/html
    environment:
      - WORDPRESS_DB_NAME=wordpress
      - WORDPRESS_TABLE_PREFIX=wp_
      - WORDPRESS_DB_HOST=mysql
      - WORDPRESS_DB_USER=uwp
      - WORDPRESS_DB_PASSWORD=123456
    links:
      - mysql
    restart: always`

**Comment by mjstealey** [comment]
@behinam - answered in #4

**Comment by mjstealey** [comment]
## Documentation

### TL;DR

locally running over http:
<img width="80%" alt="tl;dr running site" src="https://user-images.githubusercontent.com/5332509/51803325-cc65ce80-2221-11e9-8789-6dd4e802464c.png">

**Comment by mjstealey** [comment]
# mjstealey.com example deployment

<img width="80%" alt="screen shot 2019-02-05 at 8 31 43 pm" src="https://user-images.githubusercontent.com/5332509/52315624-afe63680-2985-11e9-8ac3-ccc186e9114f.png">
<img width="80%" alt="screen shot 2019-02-05 at 8 32 59 pm" src="https://user-images.githubusercontent.com/5332509/52315625-afe63680-2985-11e9-8cb1-fd657e72ecb5.png">
<img width="80%" alt="screen shot 2019-02-05 at 8 33 13 pm" src="https://user-images.githubusercontent.com/5332509/52315626-b07ecd00-2985-11e9-8a1f-626c0a5e39cc.png">
<img width="80%" alt="screen shot 2019-02-05 at 8 33 39 pm" src="https://user-images.githubusercontent.com/5332509/52315627-b07ecd00-2985-11e9-981d-3a4ac703d143.png">
<img width="80%" alt="screen shot 2019-02-05 at 8 33 50 pm" src="https://user-images.githubusercontent.com/5332509/52315628-b07ecd00-2985-11e9-82a8-9057143610fc.png">
<img width="80%" alt="screen shot 2019-02-05 at 8 34 01 pm" src="https://user-images.githubusercontent.com/5332509/52315629-b1176380-2985-11e9-9994-506d350a1237.png">

Certificate information

<img width="80%" alt="screen shot 2019-02-05 at 8 38 32 pm" src="https://user-images.githubusercontent.com/5332509/52315703-1c613580-2986-11e9-950b-092c4a5df924.png">

**Comment by mjstealey** [comment]
no longer needed
